index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="/CSS/style.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pet Care</title>
+</head>
+<body>
+    <div class="header">
+        <div class="company_logo_container"><img class="img_logo" src="Assets/images/company_logo.jpg" alt="Company Logo" ></div>
+        <h1>Pet Care</h1>
+        <div class="sandwich-menu">
+            <img src="/Assets/images/cat-logo.webp" alt="">
+        </div>
+    </div>
+
+    <div>
+        <ul class="menu">
+            <p class="menu-elements"><a href="index.html"><strong>Home</strong></a></p>
+            <p class="menu-elements"><a href="how_to.html"><strong>How to</strong></a></p>
+            <p class="menu-elements"><a href="pet_accesories.html"><strong>Pet Accesories</strong></a></p>
+            <p class="menu-elements"><a href="about.html"><strong>About</strong></a></p>
+        </ul>
+    </div>
+
+    <p class="p_title"><strong>Discover the Ultimate Pet Care Experience</strong></p>
+
+    <div class="content">
+        <img src="Assets/images/gif1-500x.gif" alt="GIF">
+    </div>
+
+    <div class="p_intro">
```

Comment:
Naming convention?
You have a "p_title" class which is applied to a p element. 
Then you have a "p_intro" class which is NOT applied to a p element, rather a div.

Reply:
so somenthing like <div class="intro_page"> would make more sense? 
The content can be change in the future and would still make sense having the class named like that.